benches/bench_redirect_performance.rs

```diff
@@ -6,6 +6,9 @@ use adblock::filters::network::{NetworkFilter, NetworkFilterMask, NetworkFilterM
 use adblock::request::Request;
 use adblock::resources::ResourceStorage;
 
+#[cfg(not(feature = "resource-assembler"))]
+use base64::prelude::*;
+
```

Comment:
this could be done inside the existing `cfg` block, next to the other `use` statement there